src/builder.rs

```diff
@@ -26,13 +28,13 @@ impl Builder {
     }
 
     #[cfg(feature = "v3_10")]
-    pub fn new_from_file(file_name: &str) -> Builder {
+    pub fn new_from_file(file_path: &Path) -> Builder {
         assert_initialized_main_thread!();
-        unsafe { from_glib_full(ffi::gtk_builder_new_from_file(file_name.to_glib_none().0)) }
+        unsafe { from_glib_full(ffi::gtk_builder_new_from_file(file_path.to_glib_none().0)) }
     }
 
     #[cfg(feature = "v3_10")]
-    pub fn new_from_resource(resource_path: &str) -> Builder {
+    pub fn new_from_resource(resource_path: &Path) -> Builder {
```

Comment:
I don't see why it couldn't. The description says it's a path to a resource file after all. What would be the difference with another file path?